custom_components/lennoxs30/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "config": {
+    "title": "Lennox S30 Local/Cloud",
+    "step": {
+      "user": {
+        "title": "Define your Lennox S30 connection",
+        "data": {
+			"name": "Name you want to use for this copy of the intergration",
+			"email": "Your email, or \"Local\" if using local lan",
+			"password": "Your password, or \"Local\" if using local lan",
+			"ip_address": "The Local IP or Hostname of your Lennox S30, or \"None\" if using cloud",
```

Comment:
If no scan interval is defined for local we use a zero scan interval which works fine.  The message retrieval now uses a long polling which causes the S30s to wait for up to 15 seconds for a message to be available before returning 204.  Hence the delay in the loop is no longer needed.